src/components/common/slider-range/slider-range.vue

```diff
@@ -19,30 +36,49 @@ function onChangeMin(value: number, dragging: boolean) {
 function onChangeMax(value: number, dragging: boolean) {
   emit('change', props.selectedMinValue, value, dragging)
 }
+
+function onChange(value: number) {
+  if (maxValue.value === undefined) {
+    minValue.value = value
+  } else {
+    if (value >= maxValue.value) {
+      maxValue.value = value
+    } else if (value <= props.selectedMinValue) {
+      minValue.value = value
+    } else if (+(maxValue.value - value) < +(minValue.value - value)) {
+      maxValue.value = value
+    } else {
+      maxValue.value = value
```

Comment:
Shouldn't this be `minValue` to move the left slider if the click is closer to it?

Reply:
You are right, thx!